### ISSUE TITLE
Can I add an additional hardware card to my existing key?

### ISSUE THREAD

**Issue by PeterTGarner** [open-keychain/open-keychain#2565]
OpenKeychain detected and set up my Yubikey 5 NFC on my phone without problems. I now want to add an additional hardware card (same phone): will OpenKeychain detect and transfer my existing RSA key to the new card?

**Comment by CueHD** [comment]
I don't think that OpenKeychain is able to transfer keys to cards. I could be wrong, but I think that it can only create links to cards that already have keys on them.
